src/Native/Runtime/gcdump.cpp

```diff
@@ -256,6 +256,8 @@ void GCDump::DumpCallsiteString(UInt32 callsiteOffset, PTR_UInt8 pbCallsiteStrin
                 if (b & CSR_MASK_X26) { gcPrintf("X26 "); count++; }
                 if (b & CSR_MASK_X27) { gcPrintf("X27 "); count++; }
                 if (b & CSR_MASK_X28) { gcPrintf("X28 "); count++; }
+#elif defined (_TARGET_WASM_)
```

Comment:
Include this file for the portable version of the runtime only instead.

Reply:
Done (assuming you meant to put it in the full runtime, not the portable one)

Reply:
Done